src/datasources.js

```diff
@@ -31,7 +102,8 @@ define([
                     dataType: 'json',
                     success: function (data) { callback(data); },
                     // TODO error handling
-                    data: {}
+                    data: {},
+                    async: false
```

Comment:
Made this change so that when itemset widget is loaded it will wait on the data source to be loaded